--- mitlm-0.4.1/src/NgramModel.cpp	2015-12-15 16:25:41.000000000 -0800
+++ mitlm-0.4.1/src/NgramModel_new.cpp	2015-12-15 16:26:50.000000000 -0800
@@ -322,7 +322,7 @@
             // Set probability and backoff weight.
             if (index == Vocab::EndOfSentence && o == 1) {
                 if (strcmp(token, "<s>") == 0) {
-                    assert(prob <= std::pow(10, -99));
+                    assert(prob <= std::pow((float)(10), (float)(-99)));
                     bows[index] = (p >= &line[lineLen]) ?
                         (Prob)1 : (Prob)std::pow(10.0, strtod(p, &p));
                 } else {
